Trikot.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name          = "Trikot"
-  spec.version       = "1.0.0"
+  spec.version       = "3.2.0-M1"
```

Comment:
What's that?

Reply:
I was triggered when installing pods, where the Podspec version would not expose the actual version of the lib 😅 : 
```
Installing Trikot (1.0.0)
```

I feel like we should instead see :
```
Installing Trikot (3.2.0-M1)
```

Reply:
maybe we could leverage the podspec_versions script and use the properties file version as a single source of truth?

Reply:
I was referring to this one
```
def load_properties(properties_filename)
  properties = {}
  File.open(properties_filename, 'r') do |properties_file|
    properties_file.read.each_line do |line|
      line.strip!
      if (line[0] != ?# and line[0] != ?=)
        i = line.index('=')
        if (i)
          properties[line[0..i - 1].strip] = line[i + 1..-1].strip
        else
          properties[line] = ''
        end
      end
    end
  end
  properties
end

```

Reply:
Actually we should migrate to the [Gradle Cocoapods Plugin](https://plugins.gradle.org/plugin/org.jetbrains.kotlin.native.cocoapods) and the versionning will be free! 😄